docs/_guide/02-writing-templates.md

```diff
@@ -286,4 +286,83 @@ html`${cache(data.showDetails
 When lit-html re-renders a template, it only updates the modified portions: it doesn't create or remove any more DOM than it needs to. But when you switch from one template to another, lit-html needs to remove the old DOM and render a new DOM tree. 
 
 The `cache` directive caches the generated DOM for a given binding and input template. In the example above, it would cache the DOM for both the  `summaryView` and `detailView` templates. When you switch from one view to another, lit-html just needs to swap in the cached version of the new view, and update it with the latest data.
+### Rendering nothing
+Very often you may conditionally render something, but in the other case
```

Comment:
This intro sort of says the same thing twice. Suggest:

Sometimes, you may want to render nothing at all. For example, you might want to render nothing when a given condition is false.

The values `undefined`, `null` and the empty string (`''`) in a text binding all render an empty text node. In most cases, that's exactly what you want.

In some cases you may need to ensure that a binding does not produce a text node at all. This is true when dealing with shadow DOM slots that have fallback content.

Reply:
... and then we can skip straight to the section on slots

Reply:
I did stick to your wording here.

Reply:
Used your text suggestion.